Jenkinsfile

```diff
@@ -1104,7 +1104,13 @@ pipeline {
                             sconsBuild(parallel_build: true,
                                        scons_args: 'PREFIX=/opt/daos TARGET_TYPE=release BUILD_TYPE=debug',
                                        build_deps: 'no'))
-                        unstash('nltr')
+                        script {
+                            try {
+                                unstash('nltr')
+                            } catch (e) {
+                                print 'Unstash failed, results from NLT stage will not be included'
+                            }
```

Comment:
You can move this code in to the Map nlt_test() function at around line 74 in the Jenkinsfile.
Then do not need to add the script block to this stage.

From what I have been told, code in the nlt_test function does not contribute to the size limitation of the Jenkinsfile.

Reply:
I've tried that in the latest push, thank you.